samples/bluetooth/central_otc/src/main.c

```diff
@@ -151,17 +157,31 @@ static void otc_btn_work_fn(struct k_work *work)
 			err = bt_ots_client_read_object_data(&otc, default_conn);
 			if (err != 0) {
 				printk("Failed to read object %d\n", err);
-				return;
 			}
 		} else {
 			printk("This OBJ does not support READ OP\n");
 		}
 	}
 }
 
+static void otc_crc_work_fn(struct k_work *work)
```

Comment:
Ditto w.r.t. using `crc` vs `checksum`; suggest to use `checksum`

Reply:
Change to ``checksum``.